### PR TITLE
add ephemeral disk support

### PR DESCRIPTION
**Pull request by naikajah**

### CODE REVIEW

File: templates/vars/aks/sbox.json [hunk 1]
```diff
@@ -19,6 +19,12 @@
     },
     "type": {
       "value": "VirtualMachineScaleSets"
+    },
+    "osDiskType": {
+      "value": "Ephemeral"
+    },
+    "vmSize": {
```
Comment (timja) [inline templates/vars/aks/sbox.json:26]: should be removed from variable group too?

Comment (naikajah) [reply templates/vars/aks/sbox.json:26]: yeah, agree.

**Review by adusumillipraveen — APPROVED** [review]

File: templates/arm-aks-template.json [hunk 2]
```diff
@@ -123,7 +127,7 @@
     {
       "type": "Microsoft.ContainerService/managedClusters",
```
Comment (avi-hmcts) [inline templates/arm-aks-template.json:128]: doesn't it have to have be "type": "Microsoft.ContainerService/EnableEphemeralOSDiskPreview"
```suggestion
      "type": "Microsoft.ContainerService/managedClusters",
```

**Review by timja — APPROVED** [review]

**Comment by thomast1906** [comment]
Closing old PR - ephemeral disk were added at a later PR

**Pull request closed without merging**